### ISSUE TITLE
CRL platform pages and clean-up

### ISSUE THREAD

**Issue by Afollet** [PDCMFinder/pdxfinder#324]
On examination of CRL many of the new datasets do not have Platform Word Press pages and the links go 404 pages. As CRL is the PDX Finder biggest contributor by care should be taken to clean this up.

See: http://dev.pdxfinder.org/data/pdx/CRL/CRL-2029#variation

Additionally, there is a duplicated model Id in the CRL metadata: CRL-2533-subrenal

This model Id has two rows with different information.

####### ACCEPTANCE CRITERIA #############
- [x] CRL has been contacted for WP content
- [ ] "ILLUMINA" platform page as been made
- [ ] "Affymetrix HGU133plus2" WP page as been made
- [ ] "RNA sequencing" WP page as been made
- [ ] "Histology" WP page has been made

**Comment by zperova** [comment]
@Afollet could you please check in the metadata who is the contact person for this dataset and email it to me? Thank you!

**Comment by Afollet** [comment]
tumor-model-compendium@crl.com | Julia Schueler, Hagen Klett

**Comment by Afollet** [comment]
Affymetrix and "Illumina" expression data was added by Csaba on 2020-06-11
"RNA sequencing" was added by Csaba on 2020-05-20

**Comment by zperova** [comment]
Great, thanks @Afollet  I contacted the provider

**Comment by zperova** [comment]
this is blocked by the provider

**Comment by CsabaHalmagyi** [comment]
Won't do, retiring wordpress.
